spec/std/yaml/yaml_pull_parser_spec.cr

```diff
@@ -95,5 +106,11 @@ module YAML
         end
       end
     end
+
+    assert_raw %(hello)
+    assert_raw %("hello"), %(hello)
+    assert_raw %(["hello"])
+    assert_raw %(["hello","world"])
+    assert_raw %({"hello":"world"})
```

Comment:
I mean, the pull parser receives events, "mapping start" and "sequence start", it doesn't matter in what form you wrote the YAML. That's tested in the regular YAML pull parser specs.